sass/_PrintableUri.scss

```diff
@@ -20,24 +20,24 @@
   .coveo-hidden {
     display: none;
   }
-  .coveo-printable-uri-part:nth-of-type(1) {
-    @include flex-shrink(1);
-    min-width: 15px;
-  }
-  .coveo-printable-uri-part:last-of-type {
-    @include flex-shrink(1);
-    min-width: 15px;
-  }
   .coveo-printable-uri-part {
     color: $color-coveo-for-salesforce;
     margin: 0;
     flex: 0 15 auto;
     min-width: 10px;
+    @supports (max-width: max-content) {
+      flex: 1 0 0;
+      max-width: max-content;
+    }
```

Comment:
This doesn't do anything on IE11.

On other browsers, this makes it so that longer elements are ellipsed first.